content/en/docs/setup/install/multicluster/shared-gateways/index.md

```diff
@@ -189,6 +190,7 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
       --set values.global.remoteTelemetryAddress=${LOCAL_GW_ADDR} \
       --set values.gateways.istio-ingressgateway.env.ISTIO_META_NETWORK="network2" \
       --set values.global.network="network2" \
+      --set values.global.multicluster.clusterName=${CLUSTER_NAME} \
```

Comment:
Make this: `--set values.global.multiCluster.clusterName=${CLUSTER_NAME}` (uppercase the C in multiCluster).

Reply:
ok, fixed it.